app/assets/javascripts/application.js

```diff
@@ -33,7 +33,33 @@ $(function() {
 });
 
 function executeAttachinary() {
+  var js_template = [
+    "<ul class='attachinary_images_list'>",
```

Comment:
I was about to ask this to @talyssonoc. There is a app/assets/javascript/templates folder with .jst.ejs files. Is there an easy way to embed one file from there into this variable somehow? Then I'd put this snippet into a .jst.ejs file as well.